runtime/altair/src/migrations.rs

```diff
@@ -533,16 +542,51 @@ mod ausd_to_foreign {
 			);
 
 			TotalIssuance::<T>::remove(DEPRECATED_AUSD_CURRENCY_ID);
+
+			// Remove undecodable storage entries
+			let (reads, writes) = remove_undecodable_storage_keys::<T>();
+			log::info!("{LOG_PREFIX} Removed {writes} undecodable storage entries from Accounts");
```

Comment:
We have at least one deprecated `CurrencyId` variant which cannot be decoded. Looking through the history, this can only be something without value, e.g. `Usd` or `KUSD` (#811).

Reply:
Thanks a lot for digging into it! Love to see a clean state 🧹